tasks/service-runner/app/workers/pull-worker.ts

```diff
@@ -130,7 +150,7 @@ async function _pullAndDoWork(repeat = true): Promise<void> {
         logger.debug(`Sending response to Harmony for results of work item with id ${workItem.id} for job id ${workItem.jobID}`);
         let tries = 0;
         let complete = false;
-        while (tries < maxRetries && !complete) {
+        while (tries < maxItemUpdateRetries && !complete) {
```

Comment:
Why are you still using the manual retry logic here instead of relying on `axiosRetry`?

Reply:
The ticket was for polling specifically so I didn't want to bite too much off.